src/components/Product.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Rating from "./Rating";
+import "../style/product.css";
+
+function Product(props) {
```

Comment:
Suggestion - You can destructure the props here itself.

Reply:
done